README.md

```diff
@@ -36,6 +23,26 @@ This pipeline is doing XYZ.
     `$ pip install -r dev-requirements.txt`
     `$ vim local_config.py` # edit, edit
     `$ alembic upgrade head` # initialize database
+
+## Important note
+
+The pipeline will NOT send anything to SOLR/Metrics DB by default. You should trigger the update using a cronjob. There are two important modes:
+
+    - normal operation (`python run.py -r`) will discover all updates that happened since the last invocation
+        and will send them to the `index-records` queue; the parameter force will be False; hence only documents
+        that have both metadata, orcid claims, and non-bib data will get sent to solr
+    - busy operation (`python run.py -r -f`) will discover all updates since the last invocation; and will send
+        them to `index-records` queue and set force=True; this will force the worker to submit data to solr
+        as soon as we have any updates (only biblio metadata need be present)
+        
+ It is **imperative** that both modes of operation be used together in the 24h cycle. The normal operation will ignore some (many)
+ records - so you must call the `busy` mode at least at the end of the quiet period. The suggested mode is the following:
+ 
+  00:00-05:00 | normal mode, invoked every 5 mins
+  05:00-06:00 | catch updates, invoke forced mode
+  06:01-17:00 | --- do nothing, solr replication is not happening
+  17:01-22:00 | normal mode, invoked every 5 mins
```

Comment:
The README describes running normal mode very 5 minutes to catch updates. How many items can be queued in 5 minutes? If we can't queue all the changed records, what happens? For the initial load, we have to queue over 12 million bibcodes.